testsrc/org/mozilla/javascript/tests/es2022/NativeObjectTest.java

```diff
@@ -0,0 +1,112 @@
+package org.mozilla.javascript.tests.es2022;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.ScriptableObject;
+
+public class NativeObjectTest {
```

Comment:
1. Is there a test for not following the prototype chain?
  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn
> ```js
> Object.hasOwn(example, 'toString');         // returns false
> ```

2. How is `Symbol` handled in the spec?